core/src/main/java/org/jdbi/v3/core/Sql.java

```diff
@@ -61,29 +65,27 @@ private Sql(String sql) {
     }
 
     public static Sql of(CharSequence... tokens) {
-        return tokens == null ? new Sql("") : of(Arrays.asList(tokens));
+        return tokens == null ? EMPTY_SQL : of(Arrays.asList(tokens));
     }
 
-    public static Sql of(Collection<CharSequence> tokens) {
-        return tokens == null ? new Sql("") : new Sql(format(tokens));
+    public static Sql of(Iterable<? extends CharSequence> tokens) {
+        return tokens == null ? EMPTY_SQL : new Sql(format(tokens));
     }
 
     /**
-     * Formats an sql statement from multiple tokens.<br>
-     * Leading and trailing whitespace is removed from each token and empty tokens ignored.<br>
-     * The tokens are joined using a single blank character to create the sql string.<br>
-     * Finally, any trailing semicolons are removed from the resulting sql.
+     * Formats an sql statement from multiple tokens.<br> Leading and trailing whitespace is removed from each token and empty tokens ignored.<br> The tokens
```

Comment:
I personally find the original formatting easier to read.
Was the Javadoc perhaps auto-formatted?

Reply:
Yes; IntelliJ. I can revert that.